src/Ilúvatar/iluvatar_worker/src/main.rs

```
@@ -37,6 +38,39 @@ async fn run(server_config: WorkerConfig, tid: &TransactionId) -> Result<()> {
             .serve(addr.parse()?),
     );
 
+    match &server_config.http_server {
+        Some(c) => {
+            match c.enabled {
+                true => {
+                    info!(tid = tid, "HTTP server enabled, starting the server");
+                    let http_server = match create_http_server(
+                        &c.address,
+                        c.port,
+                        &server_config.address,
+                        server_config.port,
+                    )
+                    .await
+                    {
+                        Ok(s) => s,
+                        Err(e) => bail_error!(tid=tid, error=%e, "Error creating HTTP server on startup"),
+                    };
+                    tokio::spawn(async move {
+                        if let Err(e) = http_server.run().await {
+                            // silent error for now
```

"silent error" should never be a thing. Log it